crates/runtime/src/actor/module/wasmbus.rs

```diff
@@ -147,8 +147,7 @@ fn write_bytes<T>(
 }
 
 #[instrument(skip(store, err))]
-fn set_host_error(store: &mut wasmtime::Caller<'_, super::Ctx>, err: impl ToString) {
-    let err = err.to_string();
```

Comment:
While `anyhow::Error` is `impl ToString`, it drops all context:
https://docs.rs/anyhow/1.0.75/anyhow/struct.Error.html#display-representations